gui/wxpython/startup/guiutils.py

```diff
@@ -218,7 +218,7 @@ def create_location_interactively(guiparent, grassdb):
     """
```

Comment:
Documentation of return value is really needed for this function.

Reply:
Yes.

Reply:
The documentation of the return value needs to go to the docstring:

```
"""Create new location using Location Wizard

Returns foo when... Returns bar when...
"""
```